### ISSUE TITLE
bin/console error message

### ISSUE THREAD

**Issue by uk3** [Runalyze/Runalyze#2269]
Hello,
at the moment, when running a bin/console command, i get the following message:
```
PHP Strict Standards:  Runalyze\Bundle\CoreBundle\Services\Configuration\ConfigurationManager and Runalyze\Bundle\CoreBundle\Services\TokenStorageAwareServiceTrait define the same property ($TokenStorage) in the composition of Runalyze\Bundle\CoreBundle\Services\Configuration\ConfigurationManager. This might be incompatible, to improve maintainability consider using accessor methods in traits instead. Class was composed in /home/user08/uk/fileserver/html/runalyze/src/CoreBundle/Services/Configuration/ConfigurationManager.php on line 78
```
sometimes console continues just showing the message, but sometimes the job exits with the message
Cheers, Ulrich.

**Comment by mipapo** [comment]
Which php version?

**Comment by laufhannes** [comment]
This is due to my last fix 25c886e05ed6f9787382c217333c64ec1ec6e665. Guess I should stop pushing things directly into master without doing more testing.

**Comment by uk3** [comment]
fails (exit 1)
```
php bin/console doctrine:cache:clear-metadata
```

works, but shows the error
```
php bin/console cache:clear --env=prod
```

shows no error
```
php bin/console runalyze:queue:consume --env=prod --stop-when-empty -vvv
```

[uk@login html]$ php -version
PHP 5.6.29 (cli) (built: Dec  8 2016 08:51:50)
